writers/streamingbatchwriter/streamingbatchwriter.go

```diff
@@ -158,9 +158,9 @@ func (w *StreamingBatchWriter) stopWorkers() {
 	}
 	w.workersWaitGroup.Wait()
 
-	w.insertWorkers = make(map[string]*streamingWorkerManager[*message.WriteInsert])
-	w.migrateWorker = nil
-	w.deleteWorker = nil
+	w.insertWorkers = nil
```

Comment:
Setting the map to `nil` so that if there's an Insert after `Close()` we'll get a nil-map panic.